components/google_docs/sources/new-document-created/new-document-created.mjs

```diff
@@ -0,0 +1,38 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "google_docs-new-document-created",
+  name: "New Document Created (Instant)",
```

Comment:
Should these sources be labeled as instant? Aren't they timer-based?

Also, can we include docs links in the descriptions?

Reply:
- These sources are subscription based. They inherit from Google Drive, like I mentioned in the sheet. [new-files-instant](https://github.com/PipedreamHQ/pipedream/blob/master/components/google_drive/sources/new-files-instant/new-files-instant.mjs). There is a timer though to renew the subscription.
- Added doc links!